docs/html/user_guide.rst

```diff
@@ -782,6 +782,49 @@ This is now covered in :doc:`../topics/repeatable-installs`.
 
 This is now covered in :doc:`../topics/dependency-resolution`.
 
+.. _`Managing a different Python interpreter`:
+
+Managing a different Python interpreter
+=======================================
```

Comment:
Can we move this into a dedicated file, as part of topics/ instead? 🙈 

I've been actively working to reduce the size of this document (#9475), so that this page is _not_ treated as a one-stop dumping ground for the content about how pip behaves/how to use it.

Reply:
Sure. I mostly just looked for where this information fitted alongside what's already there.

It's pretty short compared to the other files in "topics". I assume that's not an issue? Also, do the files in "topics" have to be in Markdown? If so, can you point me to a Markdown example of the `..tab: ... ..code-block:` construct that I can copy? I'm not clear how nested fenced blocks would work in Markdown...

Reply:
I'd prefer if we use Markdown but we don't *have* to.

If you're showing the use of `pip`, use the `pip-cli` directive -- which is only available in Markdown.

````
```{pip-cli}
$ pip install foobar
```
````

That'll convert into tabs, with Windows/Unix usage.

Otherwise, the Mark-down files use 4 backticks (instead of indentation) to nest directives.

``````
````{tab}
```{code-block} python
....
```
````

````{tab}
```{code-block} console
...
```
````
``````

https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#nesting-directives

Reply:
> It's pretty short compared to the other files in "topics". I assume that's not an issue?

No, I don't think so.